client/deposits/instant-deposits/index.js

```diff
@@ -14,32 +14,45 @@ import './style.scss';
 import InstantDepositModal from './modal';
 import { useInstantDeposit } from 'data';
 
-const InstantDepositButton = ( {
-	balance: { amount, fee, net, transaction_ids: transactionIds },
-} ) => {
+const isButtonDisabled = ( instantBalance ) => {
+	let buttonDisabled = false;
+	if (
+		0 === instantBalance.amount ||
```

Comment:
Let's just keep this check to `if( 0 === instantBalance.amount )`. I wouldn't be surprised if we offered a 0% fee in the future for some lucky merchants, and that would break this logic.

Reply:
Also, this function can be simplified to:
```
const isButtonDisabled = ( { amount } ) => 0 === amount;
```
I'm a sucker for modern JS one-liners, so feel free to ignore this comment :)

Reply:
> Let's just keep this check to if( 0 === instantBalance.amount ). I wouldn't be surprised if we offered a 0% fee in the future for some lucky merchants, and that would break this logic.

Updated in https://github.com/Automattic/woocommerce-payments/pull/1673/commits/a9112987157bfa8f0941636c970cf04024e627aa

> I'm a sucker for modern JS one-liners, so feel free to ignore this comment :)

Ignored 😄